libkbfs/folder_branch_ops.go

```diff
@@ -1792,6 +1794,10 @@ func (fbo *folderBranchOps) getMDForRekeyWriteLocked(
 		return nil, kbfscrypto.VerifyingKey{}, false, err
 	}
 
+	if md.TypeForKeying() == tlf.TeamKeying {
+		return nil, kbfscrypto.VerifyingKey{}, false, err
```

Comment:
`err` should be `nil` here just to be explicit.

Reply:
Oops, that was intended, thanks.